services/clamav/src/index.test.js

```diff
@@ -1,17 +1,24 @@
 const uploadAV = require("./index");
 
 describe("clamav", () => {
-  it("should scan uploaded document with negative result", async () => {
```

Comment:
This removes an unhappy path, does it need to be replaced?

Reply:
it hasn't actually been replaced, if you look at line 15 on the changed file you'll see it's moved further down.